betterproto/__init__.py

```diff
@@ -14,9 +14,9 @@
     Dict,
     Generator,
     Iterable,
-    List,
+    Iterator, List,
```

Comment:
need to run black

Reply:
Yes, will do!

Reply:
Black makes a change or two to `plugin.py`, which I did not include in my commit/branch (since AFAIK it wasn't related to changes I made).